packages/ord-connect/src/lib/signMessage.ts

```diff
@@ -0,0 +1,40 @@
+import { ordit } from "@sadoprotocol/ordit-sdk";
+import { Network, Wallet } from "../providers/OrdContext.tsx";
+
+interface SignMessageParams {
+  message: string;
+  wallet: Wallet;
+  address: string | Record<string, string>;
+  network: Network;
+}
+
+export default async function getSignedMessage({
+  message,
+  wallet,
+  address,
+  network,
+}: SignMessageParams) {
```

Comment:
Since it's a library from each we will be expecting specific response, I think we should force returned type here for any future new wallets to evade potential bugs. So `}: SignMessageParams): string {` (assuming that it's string result in both cases below).

Reply:
returned !